easybuild/easyconfigs/n/NVHPC/NVHPC-20.7-GCC-9.3.0.eb

```diff
@@ -0,0 +1,43 @@
+name = 'NVHPC'
+version = '20.7'
+local_gccver = '9.3.0'
+versionsuffix = '-GCC-%s' % local_gccver
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+sources = ['nvhpc_2020_%(version_major)s%(version_minor)s_Linux_x86_64_cuda_multi.tar.gz']
+
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.34', '', ('GCCcore', local_gccver)),
+    ('CUDA', '11.0', '', SYSTEM),
```

Comment:
There's no `CUDA-11.0.eb` easyconfig file in the central repository, so this will have to be tweaked too...

Reply:
The CUDA dependency actually is optional as one _can_ use the NVHPC-shipped CUDA version. But we want to use our system CUDA module.

Maybe it's easiest to add a CUDA 11 module in the central repo?

Reply:
We already have easyconfigs for CUDA 11.0.x (see `CUDA-10.1.243.eb` at https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/c/CUDA), but this looks like some kind of a wrapper to allow in-place updating of CUDA?

Reply:
@damianam, can you help?

Reply:
I think we should simply change the PR here to use whatever CUDA 11 version is available in this repo. We have `CUDA-11.0.eb` for certain constrains and design decisions that do not apply here, so I would suggest to replace `('CUDA', '11.0', '', SYSTEM),` with `('CUDA', '11.0.2'),`

Reply:
Changed as suggested in 0a1a1a3e88.